dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -239,8 +310,12 @@ def tests(
     test_type: str,
     test_timeout: str,
     db_reset: bool,
+    run_in_parallel: bool,
+    parallelism: int,
     image_tag: Optional[str],
     mount_sources: str,
+    skip_cleanup: bool,
+    include_success_outputs: bool,
```

Comment:
I think we can remove ``limit_progress_output`` parameter altogether (from both helm_tests and testing). Once we get parallel running, the limit_progess_output should be always enabled when run in parallel and disabled when not, so we do not need another flag there.